pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-addopts= -v --durations 10 --showlocals
+addopts= -v --durations 10 --showlocals -p no:ethereum
```

Comment:
is there a long for form whatever `-p` stands for?  if so can you use it?

Reply:
don't think so: -p name               early-load given plugin (multi-allowed). To avoid
                        loading of plugins, use the `no:` prefix, e.g.
                        `no:doctest`.